packages/js/product-editor/src/components/tab/edit.tsx

```diff
@@ -39,7 +43,9 @@ export function Edit( {
 				role="tabpanel"
 				className={ classes }
 			>
-				<InnerBlocks templateLock="all" />
+				{ /* eslint-disable-next-line @typescript-eslint/ban-ts-comment */ }
+				{ /* @ts-ignore Content only template locking does exist for this property. */ }
+				<InnerBlocks templateLock="contentOnly" />
```

Comment:
I guess this change ultimately hides the highlighting of each block?

Could we potentially use `context` to set this value? so we can enable/disable this on the Editor level.
I don't think it makes sense for this to be exposed through an attribute and toggled per template.

Reply:
>Also left a suggestion on making use of context for the templateLock setting in tabs, incase we wanted to switch it on, or re-use the tabs block outside of this context.

Oh I see this in your above comment.  The template lock also gets added to the block metadata, which will not be able to consume and set its property via context.

I suppose this could be filtered if we're using server-side registration of blocks, but then we need context that lives on the server as well.  Since we don't really have a use case for this yet, maybe we should punt this until we have a need for it?

Reply:
> The template lock also gets added to the block metadata, which will not be able to consume and set its property via context.

Aaah ok, cause when I updated the above to `templateLock="all"` it looked like it brought back all the selection and highlighting again, so maybe this prop overwrites the metadata config?

I was more so thinking that if we ever want to enable it for 3PD's or for a developer helper plugin, when updating their plugin we could move this all the way up the chain to a prop on the `Editor` component, so we could toggle it within our client code.
But this is somewhat pre-emptive, and I am not entirely sure if it would benefit 3PD's yet. So we could forgo it, but it would be nice if this is the only place this gets turned on, if we do decided in the future it would be beneficial for a toggle option.